<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Login</string>
   </property>
   <widget class="QLabel" name="lblID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ID :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPW">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>PW :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editID">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPW">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>70</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>555</width>
     <height>281</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>555</width>
     <height>0</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>물품 재고</string>
    </attribute>
    <widget class="QTableWidget" name="tbInventory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>531</width>
       <height>231</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>물품ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>물품명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>물품유형</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>수량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>가격</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>업데이트 날짜</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>주문 목록</string>
    </attribute>
    <widget class="QTableWidget" name="tbOrderList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>531</width>
       <height>231</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>주문번호</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>주문자명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>물품명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>물품수량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>처리여부</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>주문시간</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>물품 주문</string>
    </attribute>
    <widget class="QTableWidget" name="tbOrder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>311</width>
       <height>231</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>물품ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>물품유형</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>물품명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>재고량</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>재고량 :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblQuantity">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblOrderQuantity">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>주문수량:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOrder">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>130</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>주문하기</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>물품명 : </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblProductName">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinQuantity">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>80</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>로그</string>
    </attribute>
    <widget class="QTableWidget" name="tbLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>531</width>
       <height>231</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>로그타입</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>로그내용</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>발생시간</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>로봇제어</string>
    </attribute>
    <widget class="QPushButton" name="btnStop">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>정지</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMove_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>이동</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHome">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>홈으로</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>로봇상태 :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblRobotStatus">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>홈포지션</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="test">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTest">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
